packages/gestalt/src/README_DELETE_TableOfContents.md

```diff
@@ -0,0 +1,105 @@
+# TableOfContents
```

Comment:
@AlbertCarreras why does this file say `DELETE`? Should I delete it?

Reply:
Yes, it's a checklist to help you make sure you looked at all the files you need